pkg/rpcnode/client.go

```diff
@@ -8,12 +8,14 @@ import (
 )
 
 type Client struct {
-	ethClient *ethclient.Client
+	ethClient         *ethclient.Client
+	fallbackETHClient *ethclient.Client
 }
 
-func NewClient(ethClient *ethclient.Client) *Client {
+func NewClient(ethClient, fallbackClient *ethclient.Client) *Client {
```

Comment:
using variadic for eth client, so user can pass arbitrary number of eth client
```suggestion
func NewClient(ethClient... *ethclient.Client) *Client {
```

Reply:
updated